plugins/connection/buildah.py

```diff
@@ -129,18 +144,23 @@ def put_file(self, in_path, out_path):
         super(Connection, self).put_file(in_path, out_path)
         display.vvv("PUT %s TO %s" % (in_path, out_path), host=self._container_id)
 
-        real_out_path = self._mount_point + to_bytes(out_path, errors='surrogate_or_strict')
-        shutil.copyfile(
-            to_bytes(in_path, errors='surrogate_or_strict'),
-            to_bytes(real_out_path, errors='surrogate_or_strict')
-        )
-        # alternatively, this can be implemented using `buildah copy`:
-        # rc, stdout, stderr = self._buildah(
-        #     "copy",
-        #     [to_bytes(in_path, errors='surrogate_or_strict'),
-        #      to_bytes(out_path, errors='surrogate_or_strict')]
+        # if put_file use buildah mount, we won't able to set correctly
```

Comment:
So it won't work with mounts at all? Then we should either to remove mounts completely and use `copy` only like in #28 
Or to find a way to use mounts with user

Reply:
I kept mount in this version, as it is still used for fetching files.

Reply:
Can you please rebase on the last version? Thanks